openapi/components/schemas/RequiresTwoFactorAuth.yaml

```diff
@@ -0,0 +1,9 @@
+title: RequiresTwoFactorAuth
+type: object
+properties:
+  requiresTwoFactorAuth:
+    type: array
+    items:
+      type: string
```

Comment:
```suggestion
    items:
      type: string
      enum:
        - totp
        - otp
```

Reply:
You are missing `emailOtp`, as mentioned in #241.

Reply:
Sadly this breaks the rust generator :/
Please refer to [this](https://github.com/vrchatapi/specification/pull/352#issuecomment-2243004300) comment on how I think its best to procede

Reply:
Nvm, its just a naming conflict. Fixed here: https://github.com/vrchatapi/specification/pull/352/commits/3553a0bd88d1fef4201419fff5375c4d66a3a13f